### ISSUE TITLE
Firecloud installation is broken with the latest setuptools==50.0.0.

### ISSUE THREAD

**Issue by DailyDreaming** [broadinstitute/fiss#147]
I'm getting this on all versions:

```
(v3nv) quokka@qcore 12:28 PM ~/git/delete/bdcat-integration-tests$ pip install --no-cache firecloud==0.16.28
Collecting firecloud==0.16.28
  Downloading firecloud-0.16.28.tar.gz (52 kB)
     |████████████████████████████████| 52 kB 960 kB/s 
    ERROR: Command errored out with exit status 1:
     command: /home/quokka/git/delete/bdcat-integration-tests/v3nv/bin/python3.8 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-a7uac05d/firecloud/setup.py'"'"'; __file__='"'"'/tmp/pip-install-a7uac05d/firecloud/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-5jopwtxi
         cwd: /tmp/pip-install-a7uac05d/firecloud/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-a7uac05d/firecloud/setup.py", line 9, in <module>
        from setuptools.command.easy_install import six, rmtree_safe, rmtree, log
    ImportError: cannot import name 'six' from 'setuptools.command.easy_install' (/home/quokka/git/delete/bdcat-integration-tests/v3nv/lib/python3.8/site-packages/setuptools/command/easy_install.py)
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

`easy_install` is deprecated and they finally removed it: https://setuptools.readthedocs.io/en/latest/easy_install.html

**Comment by DailyDreaming** [comment]
To reproduce, run:

```
pip install setuptools==50.0.0 && pip uninstall firecloud && pip install --no-cache firecloud
```

**Comment by DailyDreaming** [comment]
I've posted a fix:  #148  Hopefully someone will get some time to review it.
